packages/collector/test/tracing/opentelemetry/test.js

```diff
@@ -53,16 +56,28 @@ mochaSuiteFn('opentelemetry tests', function () {
         if (process.env.INSTANA_TEST_SKIP_INSTALLING_DEPS === 'true') {
           return;
         }
-        execSync('npm install --no-save --no-package-lock --prefix ./ ./core.tgz', {
+
+        execSync('npm install --save --prefix ./ ./core.tgz', {
```

Comment:
FYI if we dont use --save, multiple execSync commands will remove previous installations. Using package.json with save + gitignore.